dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1446,7 +1464,8 @@ public MethodInvocationNode visitMethodInvocation(MethodInvocationTree tree, Voi
             // See also BaseTypeVisitor.visitMethodInvocation and QualifierPolymorphism.annotate.
             arguments = Collections.emptyList();
         } else {
-            arguments = convertCallArguments(method, actualExprs);
+            ExecutableType methodType = (ExecutableType) TreeUtils.typeOf(tree.getMethodSelect());
```

Comment:
Should this extracted into a new `TreeUtils.typeFromUse(tree)` method, modeled after `TreeUtils.elementFromUse(tree)`?